content/en/Product Updates/release-notes-september-2024.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Release Notes: September 2024"
+linkTitle: "September 2024"
+weight: 43
+description: >-
+---
+
+{{% pageinfo %}} 
+     Explore What's New from Cobalt This Month
+{{% /pageinfo %}}
+
+### September 2024
+
+#### Updated Pentest Attestation Letters
+
+We revamped the attestation letter's style for better PDF viewing and downloading, aligning it with our current brand guidelines.
+
+{{% pageinfo color="info" %}}
```

Comment:
We can remove this part, it relates to Scan info from the previous month's notes.